nucypher/network/nodes.py

```diff
@@ -1063,16 +1062,16 @@ def validate_metadata(self,
 
         # Offline check of valid stamp signature by worker
         try:
-            self.validate_worker(verify_staking=verify_staking)
+            self.validate_worker(registry=registry)
         except self.WrongMode:
-            if not accept_federated_only:
+            if bool(registry):
                 raise
 
     def verify_node(self,
                     network_middleware,
                     certificate_filepath: str = None,
-                    accept_federated_only: bool = False,
-                    force: bool = False
+                    force: bool = False,
+                    registry: BaseContractRegistry = None,
```

Comment:
Apologies for the tiny nit, but I feel like this `registry` param should be above the `force` param.

Reply:
:heavy_check_mark:  - "promoted" the parameter.